epf/src/main/java/com/daasuu/epf/EPlayerRenderer.java

```diff
@@ -99,8 +101,14 @@ public void onSurfaceCreated(final EGLConfig config) {
         previewFilter = new GlPreviewFilter(previewTexture.getTextureTarget());
         previewFilter.setup();
 
-        Surface surface = new Surface(previewTexture.getSurfaceTexture());
-        this.simpleExoPlayer.setVideoSurface(surface);
+        final Surface surface = new Surface(previewTexture.getSurfaceTexture());
+
+        new Handler(Looper.getMainLooper()).post(new Runnable() {
```

Comment:
Is this required? It seems like a workaround?

Reply:
I refactored this, but the changes are no visible here. I don't know why. Could you please check it from change log

Reply:
I see now, this code is called from the `onSurfaceCreated` callback, which is called on a renderer thread. OK!